src/site/topic16.rst

```diff
@@ -2,8 +2,18 @@
 Topic #16 --- Exceptions
 ************************
 
-Runtime Errors
-==============
+Errors
+======
+
+* In Java, there are two classes of throwable objects
+    * A *throwable* object is one that can be thrown with a ``throw`` statement
+
+* Errors are one of the two throwable classes
+* Errors are typically for unrecoverable situations
+    * They're generally abnormal situations
+* We will not try to handle and catch Errors
+* Errors and regarded as *unchecked* exceptions for the purpose of compile-time checking of exceptions
```

Comment:
Remove ~~exceptions~~, since errors are not. They are not type checked though, that part is right.